src/utils.js

```diff
@@ -403,9 +403,9 @@ function merge(schema, form, ignore, options, readonly) {
         }
 
         //If it has a titleMap make sure to update it with latest enum and names
-        if (obj.titleMap) {
-            obj.titleMap = canonicalTitleMap(obj.schema.enumNames, obj.schema.enum);
-        }
+        // if (obj.titleMap) {
+        //     obj.titleMap = canonicalTitleMap(obj.schema.enumNames, obj.schema.enum);
+        // }
```

Comment:
Are you sure we can remove this?  What was the issue?

Reply:
this part is not presented it earlier versions of library and now, actually, it's only function is to drop everything including one of tests. I dont really know what they want to do with it in future, but now there is no reasons to use it.